subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -940,8 +940,9 @@ uint8_t ll_adv_enable(uint8_t enable)
 		 */
 		conn->llcp_terminate.node_rx.hdr.link = link;
 #else
-		conn->llcp.fex.features_used = LL_FEAT;
-		conn->llcp.fex.features_peer = 0;
+		/* Re-initialize the control procedure data structures */
```

Comment:
There is some inconsistency here; maybe we should move the init of vex.valid and terminate.reason also to ll_conn_init?

Reply:
Yeah your right about vex.valid - I'll simply remove clearing it in ll_adv_enable() as this is already done by ll_conn_init() 
Re. terminate.reason - since this is in conn context and not llcp I think it should stay for now